concepts/05 UI Components/Form/01 Getting Started with Form/80 Submit the Form.md

```diff
@@ -0,0 +1,268 @@
+To submit a form, add the [Button Item](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/ButtonItem/) and set the [useSubmitBehavior](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#useSubmitBehavior) property to **true**. The Form can be submitted to a server only when all the validation checks pass successfully. 
+
+In this tutorial, we use the <a href="https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout" target="_blank">setTimeout</a> function to emulate the form submission. The Form is wrapped in the HTML <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form" target="_blank">form</a> element. We set the <a href="https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault" target="_blank">preventDefault</a> property to **true** to override the HTML form submit event:
```

Comment:
I don't understand why we want to show this here. Users will not need this in their applications, right?

Reply:
The reason is quite simple. The step when we submit the form is extremely important, and we have to show how to implement it. However, we cannot leave this event without something that indicates that we have successfully submitted the form. The easiest way to do this is to show alert window and emulate the delay that is usually caused by a server round trip. 

If you mean the second part about override - it is obligatory in the scenario when we use the useSubmitBehavior property.

Reply:
It's just looks like we want people to read/learn/copy code that is only there to fake something. Why do they need to spend their time on that? It's probably better to skip this part if we cannot show the real thing. Or explain why we cannot show it. 

If it's a tutorial, I should follow the steps. But this step doesn't really help me in any way, as I see it. 

I mean you make a link to setTimeout and preventDefault - it's like you want people to learn about them, like they'll need to use those same things. So if you want to keep that part, maybe we need to minimize it:

  ```suggestion
The code below does show how to add a Submit button, but disables its default behavior. (To keep this tutorial simple, we didn't implement the backend, so there's no way to post data to a server.)  The example simply shows a fake confirmation message after a timeout:
```

Maybe something's not right in there, but you get the idea - feel free to update.